cloud/wfm/src/result/UrlResult.ts

```diff
@@ -0,0 +1,18 @@
+// NOTE: browserify can provide an implementation for node's url module
+import * as nodeUrl from 'url';
+import {Result} from './index';
+
+/**
+ * Represents a {@link Result} that can be accessed via a URL
+ */
+export default class UrlResult implements Result {
```

Comment:
Actually I think this one is very flexible, since most results could be represented by an URL.

Moving forward I see we needing things like `AppFormsResult`, `ImageResult` (with an accompanying frontend component rendering pictures on the mobile app), and so on, but most of those would still be able to be represented via a link the user could follow to download the resource from a simple http endpoint.